test/rai/pipeline_boston_analyse.yaml

```diff
@@ -0,0 +1,102 @@
+name: AML_RAI_Analysis_VERSION_REPLACEMENT_STRING
+experiment_name: AML_RAI_Boston_VERSION_REPLACEMENT_STRING
+type: pipeline_job
+
+inputs:
+  target_column_name: 'y'
+  my_training_data:
+    data: azureml:Boston_Train_PQ
+    mode: mount
+  my_test_data:
+    data: azureml:Boston_Test_PQ
+    mode: mount
+
+outputs:
+  my_model_directory:
+    data:
+      datastore: azureml:workspaceblobstore
+  ma_info:
+    data:
+      datastore: azureml:workspaceblobstore
+  model_info:
+    data:
+      datastore: azureml:workspaceblobstore
+
+defaults:
+  component_job:
+    datastore: azureml:workspaceblobstore
+    compute:
+      target: azureml:cpucluster
+    environment: azureml:AML-RAI-Environment:VERSION_REPLACEMENT_STRING
+
+jobs:
+  train-model-job:
+    type: component_job
+    component: azureml:TrainBostonForRAI:VERSION_REPLACEMENT_STRING
+    inputs:
+      training_data: ${{inputs.my_training_data}}
+      target_column_name: ${{inputs.target_column_name}}
+      categorical_features: '[]'
+      continuous_features: '["CRIM", "ZN", "INDUS", "CHAS", "NOX", "RM", "AGE","DIS", "RAD", "TAX", "PTRATIO", "B", "LSTAT"]'
```

Comment:
should this be explicitely passed by the user? or could user not pass anything and default value be []